rtgui/gradient.cc

```diff
@@ -579,6 +580,19 @@ bool Gradient::drag1(int modifierKey)
     return false;
 }
 
+void Gradient::releaseEdit()
+{
+    if (lastObject >= 0) {
+        if (lastObject == 2 || lastObject == 3) {
```

Comment:
Magic numbers. Can't their meaning be expressed through an `enum` value?

Reply:
Replaced with enum values.